console/src/Globals.ts

```diff
@@ -38,7 +62,7 @@ const globals = {
   telemetryTopic: isProduction ? 'console' : 'console_test',
   assetsPath: window.__env.assetsPath,
   serverVersion: window.__env.serverVersion,
-  consoleAssetVersion: CONSOLE_ASSET_VERSION, // set during console build
+  consoleAssetVersion: window.CONSOLE_ASSET_VERSION, // set during console build
```

Comment:
We can't do this, as `CONSOLE_ASSET_VERSION` is not in `window` object. It's created with Webpack [DefinePlugin](https://webpack.js.org/plugins/define-plugin/) in `dev.config.js` and `prod.config.js`.

We should add `"globals"` entry to typescript overrides in `.eslintrc`:
```json
"globals": {
        "CONSOLE_ASSET_VERSION": true
      },
```

And `const CONSOLE_ASSET_VERSION: string;` to `global` declaration.

Reply:
Done! Thanks for explanation ;)